spec/fundamentals.mediawiki

```diff
@@ -15,75 +37,120 @@ This section describes each of these design aspects.
+
+Every asset is assigned a unique integer '''ID''' (<code>id</code>) that will be used to identify the
+asset in serialized structures. Whenever possible, the ID is the same as the
+BIP-0044 registered [https://github.com/satoshilabs/slips/blob/master/slip-0044.md coin type index]
+[[references.mediawiki|<nowiki>[6]</nowiki>]]. The '''symbol''' (<code>symbol</code>) is also found there.
+
+The '''swap confirmations''' (<code>swapconf</code>) is the number of confirmations required during
+settlement on the first swap transaction, before the second swap transaction is
+broadcast, as well as on the second swap transaction, before the first
+redemption is broadcast.
+
+The '''max fee rate''' (<code>maxfeerate</code>) is the greatest fee the server
+may expect the client to pay on initiation transactions.
+
+The '''unit info''' (<code>unitinfo</code>) contains information on unit names
+and includes the following variables:
+
+* The '''atomic unit''' (<code>atomicUnit</code>) is the smallest unit the asset package will use in communications. i.e. Sats for btc and gwei for eth.
+
+* The '''conventional denomination''' (<code>conventional</code>) is the most commonly used denomination '''unit''' (<code>unit</code>) and its '''conversion factor''' (<code>conversionFactor</code>). i.e. BTC at 1e8 Sats or ETH at 1e9 gwei.
+
+* The '''alternatives''' (<code>denominations</code>) are other possible '''unit''' and '''conversion factor''' combinations. i.e. µBTC at 1e2 Sats. They may not exist for some assets.
+
+===Market Variables===
+
+The '''name''' (<code>name</code>) is the market's name, composed of the base
+and quote asset symbols, lower case, with a hyphen inbetween. i.e. dcr_btc.
+
+The '''base''' (<code>base</code>) and '''quote''' (<code>quote</code>) are the
+BIP-0044 registered [https://github.com/satoshilabs/slips/blob/master/slip-0044.md coin type index]
+[[references.mediawiki|<nowiki>[6]</nowiki>]] of the base and quote assets that
+make up the market.
+
+The '''lot size''' (<code>lotsize</code>) for a market serves as both
+the minimum order quantity and the order quantity increment for limit orders
+and market buy orders, which are quantified in the market's base asset.
 In particular, for ''lot size''  '''''l''''', the requested ''order quantity'',
 '''''Q''''', must satisfy
 
 <!--Q = n l, n \in \{1, 2, 3, ...\}-->
 [[File:images/lot-sizes.png]]
 
-Every asset is assigned a unique integer ID that will be used to identify the
-asset in serialized structures. Whenever possible, the ID is the same as the
-BIP-0044 registered coin type index
-[https://github.com/satoshilabs/slips/blob/master/slip-0044.md &#91;6&#93;].
-
-When the asset is the quote asset, a '''price increment''' is enforced.
-The '''rate''', '''''r''''', of a limit order must be an integer multiple of the
-''price increment'', '''''p'''''.
+For the quote asset, a '''rate step''' (<code>ratestep</code>) is enforced. The ''rate''
```

Comment:
You could write: The "rate" is in units of the quote asset, and in must be specified in integer multiples of the "rate step".

Reply:
Changed to:
```
A trade's rate ''r'' is in units of the quote asset, and must be specified in
integer multiples of the '''rate step''' (<code>ratestep</code>) shown here as ''p''.
```